### DISCUSSION TITLE
An empty "_tracks" folder is being generated and I don't know why.

### DISCUSSION THREAD

**Discussion by mckayc** [WyattBlue/auto-editor#609] [1. Help]
After updating auto-editor, I now get a "filename_tracks" folder that is generated with the XLM file (this is for Resolve). I can't see anything about this in the documentation. Is there a way I can prevent this from happening? Thanks!

**Comment by mckayc** [comment]
Any update on this? What is the folder for and why does it keep generating? Any way to run the script so it does not generate this folder?

**Comment by WyattBlue** [comment]
This is fixed in 27.1.1

**Reply by mckayc** [reply]
Thanks! I look forward to the new release!
